File: 1094.py
n = int(input())
c = 0
r = 0
s = 0

for i in range(1 , n + 1):
    x = input().split()
    a, b, = x
    if b == 'C':
        c = c + int(a)
    if b == 'R':
        r = r + int(a)
    if b == 'S':
        s = s + int(a)

total = c + r + s
pc = (c / total) * 100
pr = (r / total) * 100
ps = (s / total) * 100

print('Total: {} cobaias'.format(total))
print('Total de coelhos: {}'.format(c))
print('Total de ratos: {}'.format(r))
print('Total de sapos: {}'.format(s))
print('Percentual de coelhos: {:.2f} %'.format(pc))
print('Percentual de ratos: {:.2f} %'.format(pr))
print('Percentual de sapos: {:.2f} %'.format(ps))

File: CSharp/1094.cs
using System;
using System.Globalization;

class URI
{
    static void Main(string[] args)
    {
        CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
        int n = int.Parse(Console.ReadLine());
        int c=0; 
        int r=0;
        int s=0;

        for (int i = 0; i < n; i++)
        {
            string[] input = Console.ReadLine().Split();
            int qtd = int.Parse(input[0]);
            char x = char.Parse(input[1]);

            switch (x)
            {
                case 'C':
                    c += qtd;
                    break;
                case 'R':
                    r += qtd;
                    break;
                case 'S':
                    s += qtd;
                    break;
            }
        }

        int total = c + r + s;
        double pc=(c / (double)total)*100;
        double pr=(r / (double)total)*100;
        double ps=(s / (double)total)*100;

        Console.WriteLine($"Total: {total} cobaias");
        Console.WriteLine($"Total de coelhos: {c}");
        Console.WriteLine($"Total de ratos: {r}");
        Console.WriteLine($"Total de sapos: {s}");
        Console.WriteLine($"Percentual de coelhos: {pc.ToString("F2")} %");
        Console.WriteLine($"Percentual de ratos: {pr.ToString("F2")} %");
        Console.WriteLine($"Percentual de sapos: {ps.ToString("F2")} %");
    }
}
